### DISCUSSION TITLE
Cannot detect Vaillant VRC 720, VR 71

### DISCUSSION THREAD

**Discussion by Kifo2023** [john30/ebusd#916] [Q&A]
I am running an Vaillant aroTherm plus with sensoCOMFORT VRC 720 and VR 71 Controller.

I am starting ebusd with these parameters:

`EBUSD_OPTS="--device=192.168.188.21:9999 --scanconfig=full --receivetimeout=100000 --configpath=http://ebusd.eu/config/ --httpport=8008 --log=all:debug --lograwdata --logfile=/var/log/ebusd.log"`

ebusctl i returns:

```
version: ebusd 23.1.23.1
update check: OK
device: 192.168.188.21:9999
signal: acquired
symbol rate: 64
max symbol rate: 410
min arbitration micros: 2
max arbitration micros: 88
min symbol latency: 12
max symbol latency: 48
reconnects: 0
masters: 2
messages: 12
conditional: 0
poll: 0
update: 4
address 10: master #2
address 31: master #8, ebusd
address 36: slave #8, ebusd, scanning
```

After a while I see
`address 10: master #2
address 31: master #8, ebusd
address 36: slave #8, ebusd
address 71: master #9
`
but no details about Vaillant controllers.

Seems that devices are not found. What else can I do?

The log file is somewhat big. Download [here](http://www.kietzmann-foto.de/downloads/ebusd/).

**Comment by moinmoin-sh** [comment]
I think I see a similar behavior here #926 
Could you try to switch to USB mode and check if it works then?

**Comment by Kifo2023** [comment]
I tried with direct connection of my ebus adapter via USB to my raspberry pi 3, bit I always got only "no signal".
The ebus was connected via a 25 m cable now. Too long?

I'm now back to a short ebus cable (<1 m) and WLAN connection with my adapter. In this configuration I see now 5 Vaillant devices with the "ebusctl i" command (finally some progress!):
- the VR 71 (controller) is there, OK
- other Vaillant devices listed: HMU00, CTLV2, VWZIO (hydraulic station?), NEXT2. What might these be?
- I also have a aroTherm plus VWL 105/6A (not yet identified)
- a VRC 720 sensocomfort (not yet identified)
- a VR 921 sensoNET (not yet identified)

**Comment by Kifo2023** [comment]
I will discuss the remaining questions elswhere in this forum.
Topic closed.

**Comment by Kifo2023** [comment]
I will discuss the remaining questions elswhere in this forum.
Topic closed.
